addons/hr/tests/test_multi_company.py

```diff
@@ -26,8 +26,8 @@ def test_multi_company_report(self):
         content, content_type = self.env.ref('hr.hr_employee_print_badge').with_user(self.res_users_hr_officer).with_context(
             allowed_company_ids=[self.company_1.id, self.company_2.id]
         )._render_qweb_pdf(res_ids=self.employees.ids)
-        self.assertIn('Bidule', content)
-        self.assertIn('Machin', content)
+        self.assertIn('Bidule', str(content))
+        self.assertIn('Machin', str(content))
```

Comment:
That's not really correct. If `_render_qweb_pdf` returns `bytes` then these should be decoded, not str'd/repr'd.

Reply:
Done (needles bytes)